Makefile

```diff
@@ -1,4 +1,9 @@
-.PHONY: local-up local-down local-logs clean-docker-buildcache clean-docker-images clean-docker-containers start-lumigator-external-services start-lumigator stop-lumigator
+.PHONY: local-up local-down local-logs \
```

Comment:
not sure if it's just Github rendering but we'll want to reformat so it matches the old PHONY (without line breaks) for legibility

Reply:
Ok. The PHONY was getting so large I decided to break it up... :-/